renode-linux-runner-docker/payload/run-in-renode.py

```diff
@@ -48,6 +50,175 @@ def __getattr__(self, attr):
         return getattr(self.stream, attr)
 
 
+class Action(Protocol):
+    """
+    Action type used by the add_devices function. It works on certain
+    command when command semantic requires additional action.
+    """
+    def __call__(self, *args: Any) -> str:
+        raise NotImplementedError
+
+
+class GPIO_Duplicate_Line:
```

Comment:
I think this should be named something like `GPIO_SplitDevice`? Doesn't really duplicate lines

Reply:
You're right. This is a holdover from the previous implementation.